tests/test-cmd/test.run.js

```diff
@@ -10,7 +10,7 @@ import {onlyInstancesOf, WebExtError, RemoteTempInstallNotSupported}
 import run, {
   defaultFirefoxClient, defaultWatcherCreator, defaultReloadStrategy,
 } from '../../src/cmd/run';
-import * as firefox from '../../src/firefox';
+import * as firefoxApp from '../../src/firefox';
```

Comment:
@shubheksha don't worry, asking question is a good thing.

The very minimal change that I was suggesting require two additional identifier renamed in the test file:
```
diff --git a/tests/test-cmd/test.run.js b/tests/test-cmd/test.run.js
index 49dbd57..7293530 100644
--- a/tests/test-cmd/test.run.js
+++ b/tests/test-cmd/test.run.js
@@ -10,7 +10,7 @@ import {onlyInstancesOf, WebExtError, RemoteTempInstallNotSupported}
 import run, {
   defaultFirefoxClient, defaultWatcherCreator, defaultReloadStrategy,
 } from '../../src/cmd/run';
-import * as firefoxApp from '../../src/firefox';
+import * as defaultFirefox from '../../src/firefox';
 import {RemoteFirefox} from '../../src/firefox/remote';
 import {TCPConnectError, fakeFirefoxClient, makeSureItFails, fake, fixturePath}
   from '../helpers';
@@ -69,7 +69,7 @@ describe('run', () => {
       run: () => Promise.resolve(),
       ...implementations,
     };
-    return fake(firefoxApp, allImplementations);
+    return fake(defaultFirefox, allImplementations);
   }
```

Or, to rename `defaultFirefox` into `defaultFirefoxApp` (mainly because I think that `defaultFirefoxApp` immediately suggests that it is the default implementation of what we now call `firefoxApp` and not `firefox`) we need to apply some minor changes to both `"test.run.js"` and `"src/cmd/run.js"`, e.g. this seems to be the needed changes in the `"src/cmd/run.js"` (besides the slightly changed version of the above patch applied to `"test.run.js"`):

```
diff --git a/src/cmd/run.js b/src/cmd/run.js
index 15e29d9..8c06f83 100644
--- a/src/cmd/run.js
+++ b/src/cmd/run.js
@@ -1,5 +1,5 @@
 /* @flow */
-import * as defaultFirefox from '../firefox';
+import * as defaultFirefoxApp from '../firefox';
 import defaultFirefoxConnector from '../firefox/remote';
 import {
   onlyInstancesOf, onlyErrorsWithCode, RemoteTempInstallNotSupported,
@@ -153,7 +153,7 @@ export type CmdRunParams = {
 };

 export type CmdRunOptions = {
-  firefoxApp: typeof defaultFirefox,
+  firefoxApp: typeof defaultFirefoxApp,
   firefoxClient: typeof defaultFirefoxClient,
   reloadStrategy: typeof defaultReloadStrategy,
 };
@@ -164,7 +164,7 @@ export default function run(
     preInstall=false, noReload=false,
   }: CmdRunParams,
   {
-    firefoxApp=defaultFirefox,
+    firefoxApp=defaultFirefoxApp,
     firefoxClient=defaultFirefoxClient,
     reloadStrategy=defaultReloadStrategy,
   }: CmdRunOptions = {}): Promise<Object> {
@@ -276,7 +276,7 @@ export type ExtensionRunnerParams = {
   sourceDir: string,
   manifestData: ExtensionManifest,
   profilePath: string,
-  firefoxApp: typeof defaultFirefox,
+  firefoxApp: typeof defaultFirefoxApp,
   firefox: string,
 };

@@ -284,7 +284,7 @@ export class ExtensionRunner {
   sourceDir: string;
   manifestData: ExtensionManifest;
   profilePath: string;
-  firefoxApp: typeof defaultFirefox;
+  firefoxApp: typeof defaultFirefoxApp;
   firefox: string;

   constructor(
```